osu.Game.Rulesets.Osu/Difficulty/OsuDifficultyAttributes.cs

```diff
@@ -9,6 +9,7 @@ public class OsuDifficultyAttributes : DifficultyAttributes
     {
         public double AimStrain { get; set; }
         public double SpeedStrain { get; set; }
+        public double FlashlightRating { get; set; }
```

Comment:
Should we really be adding attributes based specifically on mod names? I don't know this seems like a really bad direction.

Reply:
What are you proposing? It's a special case so I think it's warranted?

Reply:
It could be VisualRating or ReadingStrain, something like that. It wouldn't make sense to keep adding ratings for each visual mod, as they all affect the same thing and they usually interact in different ways.

For example, the difference between nomod and HD is not the same as the difference between FL and FLHD.
These interactions should be accounted for when calculating the difficulty rating.